napalm/junos/junos.py

```diff
@@ -293,9 +293,12 @@ def _convert_to_dict(interfaces):
                         napalm.base.helpers.mac,
                         interfaces[iface]['mac_address'],
                         py23_compat.text_type(interfaces[iface]['mac_address'])),
-                    'speed': -1
+                    'speed': -1,
+                    'mtu': -1
                 }
                 # result[iface]['last_flapped'] = float(result[iface]['last_flapped'])
+            if interfaces[iface]['mtu'] != 'Unlimited':
```

Comment:
I think this is missing indentation as it seems to be outside the previous for loop.